cluster/ci/config/prow/plugins.yaml

```diff
@@ -578,6 +578,10 @@ config_updater:
       name: prow-job-cluster-launch-e2e-azure-upgrade
       additional_namespaces:
       - ci-stg
+    ci-operator/templates/openshift/openshift-azure/rbac.yaml:
+      name: prow-job-rbac-azure
+      additional_namespaces:
+      - ci-stg
```

Comment:
@bbguimaraes I'm going to `/approve` but please shout if this somehow breaks the staging namespace.

Reply:
That looks ok.